.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Latest Documentation
```

Comment:
Do we just want to add these steps to the `release.yml`?

Reply:
https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
Or maybe by virtue of the `on: release:` this will run along side the `Release` workflow?